src/OptimisticTokenVotingPlugin.sol

```diff
@@ -539,6 +586,26 @@ contract OptimisticTokenVotingPlugin is
         });
     }
 
+    /// @notice Internal function to update the excluded voting power holders.
+    /// @param _excludedVotingPowerHolders The list of addresses to be excluded from voting power.
+    function _updateExcludedVotingPowerHolders(address[] calldata _excludedVotingPowerHolders) internal {
+        delete excludedVotingPowerHolders;
+        address prev = address(0);
+
+        for (uint256 i = 0; i < _excludedVotingPowerHolders.length; i++) {
+            address current = _excludedVotingPowerHolders[i];
+            if (current == address(0)) revert EmptyAddress();
+            if (current == address(votingToken)) revert VotingTokenAddressNotAllowed();
```

Comment:
What's the issue with votingToken being in this list?
The token contract itself can't vote. If someone decides to sent his tokens to the token contract itself (aka burn them), this would still be a very reasonable cose for excluding them.